src/mmw/js/src/modeling/controllers.js

```diff
@@ -114,15 +114,37 @@ var ModelingController = {
             project = App.currentProject;
             project.set('model_package', modelPackage);
             itsiResetProject(project);
+            App.getMapView().addSidebarToggleControl();
         } else {
             var lock = $.Deferred();
-            project = makeProject(modelPackage, lock);
+            project = new models.ProjectModel({
+                name: 'Untitled Project',
+                created_at: Date.now(),
+                area_of_interest: App.map.get('areaOfInterest'),
+                area_of_interest_name: App.map.get('areaOfInterestName'),
+                model_package: modelPackage,
+                scenarios: new models.ScenariosCollection()
+            });
+
             App.currentProject = project;
+            lock.resolve();
+
+            App.getAnalyzeCollection().findWhere({taskName:'analyze'}).fetchAnalysisIfNeeded()
+            .done(function() {
+                App.currentProject.set(
+                    'aoi_census',
+                    JSON.parse(
+                        App.getAnalyzeCollection().findWhere({taskName:'analyze'}).get('result')
+                    ).census
+                );
+            })
+            .fail(function() { projectCleanUp(); });
```

Comment:
Prefer fully indented blocks, even when the body is a single statement:

```js
.fail(function() {
    projectCleanup();
});
```

Or even better, simply pass the handler directly rather than wrapping it in a function:

```js
.fail(projectCleanup);
```